### PR TITLE
Add cpu and traffic into performance overview

### LINKED ISSUE #18697: the cpu/memory and traffic in performance overview should be updated in doc
## Change Request

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.
performance overview grafafa is updated, while the doc is not udpated.

2. Describe your suggestion or addition.

3. Provide some reference materials (such as documents and websites) if you could.

### PR DESCRIPTION
**Pull request by dbsid**
fix #18697

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
Hi @dbsid, would you please involve a tech reviewer for this PR? Thanks.

**Review by Yui-Song — APPROVED** [review]

File: performance-tuning-methods.md [hunk 1]
```diff
@@ -216,34 +216,85 @@ In this workload, only `ANALYZE` statements are running in the cluster:
...
+[rocksdb.titan]
+enabled = true
+[rocksdb.defaultcf.titan]
+min-blob-size = "1KB"
+blob-file-compression = "zstd"
+```
+
+**Example 1: read and write traffic in TPCC workload**
+
+The read write traffic In the TPC-C workload as below:
```
Comment (qiancai) [inline performance-tuning-methods.md:263]:
```suggestion
The following is an example of read and write traffic in the TPC-C workload.

```

File: performance-tuning-methods.md [hunk 2]
```diff
@@ -216,34 +216,85 @@ In this workload, only `ANALYZE` statements are running in the cluster:
...
+- TiDB -> TiKV: general: 13.1 MB/s
+- TiDB -> TiKV: internal: 5.07 KB/s
+- TiKV -> Rocksdb: 109 MB/s
+- RocksDB Compaction: 567 MB/s
+
+![TPC-C](/media/performance/tpcc_read_write_traffic.png)
+
+**Example 2: write traffic before and after Titan**
+
+For an insert workload with a 6KB record size, Titan significantly reduces the write traffic and compaction I/O flow. Titan reduces compaction traffic significantly, which in turn boosts the overall performance and resource utilization of TiKV. Here are the changes:
```
Comment (qiancai) [inline performance-tuning-methods.md:279]:
```suggestion
 The following example shows the performance changes before and after Titan is enabled. For an insert workload with 6KB records, Titan significantly reduces write traffic and compaction I/O, enhancing TiKV's overall performance and resource utilization.
```

File: performance-tuning-methods.md [hunk 3]
```diff
@@ -216,34 +216,85 @@ In this workload, only `ANALYZE` statements are running in the cluster:
...
-**Example 2: High TiKV resource usage**
+#### Data Traffic
 
-In the TPC-C workload below, each TiDB and TiKV is configured with 16 CPUs.
+The read and write panels provide insights into the traffic patterns within your TiDB cluster. These panels help in monitoring both read and write traffic, offering a comprehensive view of data flow from clients to the database and between internal components.
 
-![TPC-C](/media/performance/tpcc_cpu_io.png)
+##### Read Traffic
+- TiDB -> Client: This metric displays the outbound traffic statistics from TiDB to the client.
+- Rocksdb -> TiKV: This metric tracks the data flow TiKV retriving from RocksDB during read operations within the storage layer.
```
Comment (qiancai) [inline performance-tuning-methods.md:244]:
```suggestion
- `TiDB -> Client`: the outbound traffic statistics from TiDB to the client
- `Rocksdb -> TiKV`: the data flow that TiKV retrieves from RocksDB during read operations within the storage layer
```

File: performance-tuning-methods.md [hunk 4]
```diff
@@ -216,34 +216,85 @@ In this workload, only `ANALYZE` statements are running in the cluster:
...
+Obviously, TiKV consumes more CPU, which is expected because TPC-C is a write-heavy scenario. It is recommended that you scale out the TiKV to improve performance.
 
-**Example 2: High TiKV resource usage**
+#### Data Traffic
 
-In the TPC-C workload below, each TiDB and TiKV is configured with 16 CPUs.
+The read and write panels provide insights into the traffic patterns within your TiDB cluster. These panels help in monitoring both read and write traffic, offering a comprehensive view of data flow from clients to the database and between internal components.
 
-![TPC-C](/media/performance/tpcc_cpu_io.png)
+##### Read Traffic
```
Comment (qiancai) [inline performance-tuning-methods.md:242]:
```suggestion
##### Read Traffic

```

File: performance-tuning-methods.md [hunk 5]
```diff
@@ -216,34 +216,85 @@ In this workload, only `ANALYZE` statements are running in the cluster:
...
+- The maximum CPU usage of PD are 59.1%. The maximum memory usage is 221MB. 
 
-Obviously, TiDB consumes more CPU, which is near the bottleneck threshold of 8 CPUs. It is recommended that you scale out the TiDB.
+Obviously, TiKV consumes more CPU, which is expected because TPC-C is a write-heavy scenario. It is recommended that you scale out the TiKV to improve performance.
 
-**Example 2: High TiKV resource usage**
+#### Data Traffic
 
-In the TPC-C workload below, each TiDB and TiKV is configured with 16 CPUs.
+The read and write panels provide insights into the traffic patterns within your TiDB cluster. These panels help in monitoring both read and write traffic, offering a comprehensive view of data flow from clients to the database and between internal components.
```
Comment (qiancai) [inline performance-tuning-methods.md:240]:
```suggestion
The read and write panels offer insights into traffic patterns within your TiDB cluster, allowing you to monitor data flow from clients to the database and between internal components comprehensively.
```

File: performance-tuning-methods.md [hunk 6]
```diff
@@ -216,34 +216,92 @@ In this workload, only `ANALYZE` statements are running in the cluster:
 - The total number of KV requests per second is 35.5 and the number of Cop requests per second is 9.3.
 - Most of the KV processing time is spent on `Cop-internal_stats`, which indicates that the most time-consuming KV request is `Cop` from internal `ANALYZE` operations.
```
Comment (qiancai) [inline performance-tuning-methods.md:218]: PR preview: https://pingcap-docsite-preview-git-pr-qiancais-projects.vercel.app/tidb/dev/performance-tuning-methods#cpu-and-memory-usage

**Review by lilin90 — APPROVED** [review]
/lgtm

File: performance-tuning-methods.md [hunk 7]
```diff
@@ -216,34 +216,92 @@ In this workload, only `ANALYZE` statements are running in the cluster:
...
+- The maximum CPU usage of PD is 59.1%. The maximum memory usage is 221 MiB.
 
-Obviously, TiDB consumes more CPU, which is near the bottleneck threshold of 8 CPUs. It is recommended that you scale out the TiDB.
+Obviously, TiKV consumes more CPU, which is expected because TPC-C is a write-heavy scenario. To improve performance, it is recommended to scale out TiKV.
 
-**Example 2: High TiKV resource usage**
+#### Data traffic
 
-In the TPC-C workload below, each TiDB and TiKV is configured with 16 CPUs.
+The read and write panels offer insights into traffic patterns within your TiDB cluster, allowing you to monitor data flow from clients to the database and between internal components comprehensively.
```
Comment (qiancai) [inline performance-tuning-methods.md:240]:
```suggestion
The read and write traffic panels offer insights into traffic patterns within your TiDB cluster, allowing you to monitor data flow from clients to the database and between internal components comprehensively.
```

File: dashboard/dashboard-monitoring.md [hunk 8]
```diff
@@ -77,50 +77,70 @@ Generally, `tso - request` divided by `tso - cmd` is the average size of TSO req
...
-### TiDB CPU
+### TiDB CPU/Memory
 
-- avg: Average CPU utilization across all TiDB instances
-- delta: Maximum CPU utilization of all TiDB instances minus minimum CPU utilization of all TiDB instances
-- max: Maximum CPU utilization across all TiDB instances
+- `CPU-Avg`: Average CPU utilization across all TiDB instances
+- `CPU-Delta`: Maximum CPU utilization of all TiDB instances minus minimum CPU utilization of all TiDB instances
+- `CPU-Max`: Maximum CPU utilization across all TiDB instances
+- `CPU-Quota`: TiDB CPU quota
```
Comment (qiancai) [inline dashboard/dashboard-monitoring.md:89]:
```suggestion
- `CPU-Quota`: the number of CPU cores that can be used by TiDB
```

File: dashboard/dashboard-monitoring.md [hunk 9]
```diff
@@ -77,50 +77,70 @@ Generally, `tso - request` divided by `tso - cmd` is the average size of TSO req
...
-- CPU-Avg: Average CPU utilization of all TiKV instances
-- CPU-Delta: Maximum CPU utilization of all TiKV instances minus minimum CPU utilization of all TiKV instances
-- CPU-MAX: Maximum CPU utilization among all TiKV instances
-- IO-Avg: Average MBps of all TiKV instances
-- IO-Delt: Maximum MBps of all TiKV instances minus minimum MBps of all TiKV instances
-- IO-MAX: Maximum MBps of all TiKV instances
+- `CPU-Avg`: Average CPU utilization across all TiKV instances
+- `CPU-Delta`: Maximum CPU utilization of all TiKV instances minus minimum CPU utilization of all TiKV instances
+- `CPU-Max`: Maximum CPU utilization across all TiKV instances
+- `CPU-Quota`: TiKV CPU quota
```
Comment (qiancai) [inline dashboard/dashboard-monitoring.md:97]:
```suggestion
- `CPU-Quota`: Number of CPU cores that can be used by TiKV
```

File: dashboard/dashboard-monitoring.md [hunk 10]
```diff
@@ -77,50 +77,70 @@ Generally, `tso - request` divided by `tso - cmd` is the average size of TSO req
...
+- `Mem-Max`: Maximum memory utilization across all TiKV instances
+
+### PD CPU/Memory
+
+- `CPU-Max`: Maximum CPU utilization across all PD instances
+- `CPU-Quota`: PD CPU quota
```
Comment (qiancai) [inline dashboard/dashboard-monitoring.md:103]:
```suggestion
- `CPU-Quota`: Number of CPU cores that can be used by PD
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**